modules/aws-emr-cluster/main.tf

```diff
@@ -25,31 +25,42 @@ resource "aws_emr_cluster" "emr-cluster" {
     key_name                          = var.key_pair_name
   }
 
-
-  master_instance_group {
-    name          = var.master_instance_group_name
-    instance_type = var.master_instance_type
-    # NOTE: value must be 1 or 3
-    instance_count = var.master_group_instance_count
-    # Spot Instance definition
-    bid_price = var.master_bid_price
-    ebs_config {
-      size                 = var.master_ebs_size
-      type                 = var.master_ebs_type
-      volumes_per_instance = var.master_ebs_volumes_count
+  master_instance_fleet {
```

Comment:
Just a curiosity question - Does your testing involve only using spot instances for core nodes?

Reply:
As of now, the test plan specifies testing spot instances for core nodes only. We may well try out spot instances for master nodes as well as testing progresses.